docs/how_to/extending_page_title.rst

```diff
@@ -200,6 +200,10 @@ page extension to every page, a page may not have the ``iconextension``
 relationship available, hence the use of the ``{% if ... %}...{% endif %}``
 above.
 
+In order to access to a title extension in page templates you will need to access the title object with the
+get_title_obj function:
```

Comment:
I would express this like this:

```
In order to access a title extension within a template, get the ``Title`` object using
``request.current_page.get_title_obj``, for example::

    {{ request.current_page.get_title_obj.your_title_extension }}
```

I would also like to see sub-headings within this section describing how to use extensions in templates:

    Page extensions
    ---------------

and 

    Title extensions
    ----------------